cores/esp8266/HardwareSerial.h

```diff
@@ -84,7 +84,7 @@ class HardwareSerial: public Stream
         begin(baud, config, mode, 1);
     }
 
-    void begin(unsigned long baud, SerialConfig config, SerialMode mode, uint8_t tx_pin);
+    void begin(unsigned long baud, SerialConfig config, SerialMode mode, uint8_t tx_pin, bool invert=false);
```

Comment:
This is a change in signature, which can be potentially breaking. Instead, prefer an overload with the new signature and forward the call with the added arg as default value.

Reply:
it is not a virtual method, so there is no problem with a new parameter with default value

Reply:
Too late now :-)

Reply:
@jandrassy if you want to discuss, look me up on gitter